pom.xml

```diff
@@ -732,7 +732,7 @@
         <junit.version>4.13.1</junit.version>
         <kafka.scala.binary.version>2.12</kafka.scala.binary.version>
         <kafka.version>2.8.1</kafka.version>
-        <keycloak.version>15.0.2.1</keycloak.version>
+        <keycloak.version>15.0.2</keycloak.version>
```

Comment:
Remove this change as well, keep it on your local if required

Reply:
Done